CMakePresets.json

```diff
@@ -0,0 +1,72 @@
+{
+  "version": 3,
+  "configurePresets": [
+    {
+      "name": "base-config",
+      "displayName": "Base config (not usable as preset)",
+      "description": "Should be inherited by some other presets",
+      "binaryDir": "${sourceDir}/out/build/${presetName}",
+      "installDir": "${sourceDir}/out/install/${presetName}",
+      "hidden": true
+    },
+    {
+      "name": "windows-base",
+      "hidden": true,
+      "inherits": [ "base-config" ],
+      "toolchainFile": "c:/dev/vcpkg/scripts/buildsystems/vcpkg.cmake",
```

Comment:
absolute path in committed file. As we discussed, not ideal. Is it really needed? The vcpkg toolchain is default in VS at least

Reply:
Line with toolchain file is now removed. We should set the CMAKE_TOOLCHAIN_FILE as an environment variable on developer machines. If we use a preset file, and toolchainFile is left out, CMAKE_TOOLCHAIN_FILE will be empty. So, set the environment variable instead